ca_drought/data/main.js.mustache

```diff
@@ -0,0 +1,441 @@
+var map;
+var timesteps = {{{timesteps}}};
```

Comment:
We should be careful in templating javascript, it can go quite wrong with how many `{` and `}` there are in the language.